releng/org.eclipse.emfcloud.modelserver.releng.p2/pom.xml

```diff
@@ -1,135 +1,134 @@
-							<repositories>
-								<repository>
-									<id>2019-09</id>
-									<layout>p2</layout>
-									<url>http://download.eclipse.org/releases/2019-09</url>
-								</repository>
-							</repositories>
-							<dependencies>
-								<dependency>
-									<artifactId>org.eclipse.ant.core</artifactId>
-									<type>eclipse-plugin</type>
-								</dependency>
-								<dependency>
-									<artifactId>org.apache.ant</artifactId>
-									<type>eclipse-plugin</type>
-								</dependency>
-								<dependency>
-									<artifactId>org.eclipse.equinox.p2.repository.tools</artifactId>
-									<type>eclipse-plugin</type>
-								</dependency>
-								<dependency>
-									<artifactId>org.eclipse.equinox.p2.core.feature</artifactId>
-									<type>eclipse-feature</type>
-								</dependency>
-								<dependency>
-									<artifactId>org.eclipse.equinox.p2.extras.feature</artifactId>
-									<type>eclipse-feature</type>
-								</dependency>
-								<dependency>
-									<artifactId>org.eclipse.equinox.ds</artifactId>
-									<type>eclipse-plugin</type>
-								</dependency>
-							</dependencies>
-						</configuration>
-						<executions>
-							<execution>
-								<id>add-p2-composite-repository</id>
-								<phase>package</phase>
-								<goals>
-									<goal>eclipse-run</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-					<!-- Skip cleaning of this module otherwise the release build is not 
-						working on the jenkins instance -->
-					<plugin>
-						<artifactId>maven-clean-plugin</artifactId>
-						<version>3.1.0</version>
-						<configuration>
-							<skip>true</skip>
-						</configuration>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
+               <plugin>
+                  <groupId>org.eclipse.tycho.extras</groupId>
+                  <artifactId>tycho-eclipserun-plugin</artifactId>
+                  <version>${tycho.version}</version>
+                  <configuration>
+                     <executionEnvironment>JavaSE-11</executionEnvironment>
+                     <!-- IMPORTANT: DO NOT split the arg line -->
+                     <appArgLine>-application org.eclipse.ant.core.antRunner -buildfile packaging-p2composite.ant p2.composite.add -Dsite.label="Eclipse EMF.Cloud Model Server Nightly Updatesite"
+                        -Dproject.build.directory=${project.build.directory} -DunqualifiedVersion=${unqualifiedVersion} -DbuildQualifier=${buildQualifier} -Dsoftware.download.area=${rsync.local.dir}</appArgLine>
+                     <repositories>
+                        <repository>
+                           <id>2019-09</id>
+                           <layout>p2</layout>
+                           <url>http://download.eclipse.org/releases/2019-09</url>
```

Comment:
I'm not sure, but should we use the `2020-09` repository here as well, as we use it in the target already?

Reply:
Yes, you are right, great idea, I updated it as you suggested!